ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2495,6 +2495,20 @@ tests:
     - ref: olmv1-disable-default-clustercatalogs
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: baremetalds-ipi-ovn-lvms-f14-disasterrecovery
+  cluster: build05
+  cron: 1 13 5,25 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      FILTERS_ADDITIONAL: ""
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.19
+      LVM_OPERATOR_SUB_SOURCE: qe-app-registry
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```

Comment:
https://github.com/openshift/release/blob/4265f8fe25dfa42d3c81b7b187366878f80320f6/ci-operator/step-registry/openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-ref.yaml#L17-L18, we really don't need them.

Reply:
@geliu2016 You can remove duplicate filters from test profile.

Reply:
updated.